ROADMAP.md

```diff
@@ -2,60 +2,47 @@
 
 ## About this Document
 
-This document provides an overview of the major themes driving Tanzu Community Edition development, as well as some of the features and
-capabilities that will manifest those priorities, grouped into 0-3 month, 3-6 month, and future buckets. Most items are gathered from
-the community or include a feedback loop with the community. This should serve as a reference point for Tanzu Community Edition users
-to help us prioritize and to supply input on unmet needs, and contributors to understand where the project is heading, and help
-contributors to determine if a contribution could be conflicting with a longer term plan.
+This document provides an overview of the major themes driving Tanzu Community Edition development, as well as constituent features and capabilities. Roadmap items are tracked by their implementation state within the enhancements proposal process: Awaiting Proposal, Proposal in Review, and Implementation in Progress. Most items are gathered from the community or include a feedback loop with the community.
 
-More detail about short-term delivery plans can be found in the [milestones](https://github.com/vmware-tanzu/community-edition/milestones) in
-the [GitHub project repo](https://github.com/vmware-tanzu/community-edition).
+This roadmap should serve as a reference point for Tanzu Community Edition users to help us prioritize existing features and provide input on unmet needs, and for contributors to understand where the project is heading. A roadmap also helps our contributor community to determine if a proposed enhancement conflicts with the long-term plan.
 
-## How to help
+Enhancement states help everyone understand if something is in the early stages of an idea or if it has wider community buy-in and committed resources. This is important because it helps limit waste, and focus work on the most important and timely activities. The stages are described below:
```

Comment:
Recommend rewording this paragraph, it reads really weird.

Also, not sure there should be a command between `limit waste` and `and focus work`?

Reply:
Agreed ^ I'm not sure what `Enhancement states` are